cmake/External/aotriton.cmake

```diff
@@ -0,0 +1,28 @@
+if(NOT __AOTRITON_INCLUDED)
```

Comment:
@malfet friendly ping, I imagine that doing this swap is fine but wanted to check that there isn't anything else we should aware of